rpc/amqp-rpc.go

```diff
@@ -407,7 +407,7 @@ func (rpc *AmqpRPCServer) Start(c cmd.Config) error {
 			select {
 			case msg, ok := <-msgs:
 				if ok {
-					rpc.processMessage(msg)
+					go rpc.processMessage(msg)
```

Comment:
per offline discussion, we'll totally do an atomic int check here, as well, and send back some backpressure error message, as well.